cmd/ctr/commands/run/run_unix.go

```diff
@@ -220,6 +220,11 @@ func NewContainer(ctx gocontext.Context, client *containerd.Client, context *cli
 		if limit != 0 {
 			opts = append(opts, oci.WithMemoryLimit(limit))
 		}
+		quota := context.Int64("cpu-quota")
+		period := context.Uint64("cpu-period")
+		if quota != -1 || period != 0 {
```

Comment:
Let's add them near the `context.Float64("cpus")` line for readability

Reply:
Also should error out when `--cpus` is specified together

Reply:
move descriptions to near the cpus, and change if cpus is used, skip applying cpu-quota and cpu-period.